### PR TITLE
Use React360 for VR support

### PR DESCRIPTION
**Pull request by kkemple**
React VR has moved to `react-360` namespace ([docs](https://facebook.github.io/react-360/docs/setup.html)). This PR just updates name of package and that works 🎉 

screen shot 2018-05-02 at 4 36 21 pm

### CODE REVIEW

**Comment by kkemple** [comment]
ugh, just saw prettier went haywire, let me know and I will remove those changes!

File: src/vr/Touchable.js [hunk 1]
```diff
@@ -146,35 +141,31 @@ const Touchable = (
...
+      Animated.timing(this.props.press, {
+        toValue,
+        duration: this.props.pressDuration,
+        // easing: Easing.inOut(Easing.quad),
+      }).start();
     },
 
-
-    touchableHandleActivePressIn: throttle(function (e) {
+    touchableHandleActivePressIn: throttle(function(e) {
```
Comment (ljharb) [inline src/vr/Touchable.js]: definitely please revert this :-) `function`, space, optional name, argument list, always

File: src/vr/Touchable.js [hunk 2]
```diff
@@ -18,11 +18,11 @@ const THROTTLE_MS = 500;
 function throttle(fn, throttleMs) {
   let lastCall = null;
 
-  return function (...args) {
-    const now = new Date;
-    if (lastCall === null || (now - lastCall > throttleMs)) {
+  return function(...args) {
+    const now = new Date();
```
Comment (ljharb) [inline src/vr/Touchable.js]: the parens here on the `new` tho, can stay <3

**Comment by kkemple** [comment]
sorry for the delay on this, updated!

**Comment by kkemple** [comment]
there was no dependency for `react-vr` in package.json, i assumed that was intentional

**Comment by ljharb** [comment]
You’re probably right. Should the file be renamed?

**Comment by kkemple** [comment]
Oh, that's a great question, I think `vr` still makes sense but could see it being called `react-360` or something, did you have a name in mind?

**Comment by ljharb** [comment]
i think having it match the package name makes the most sense.

**Comment by ljharb** [comment]
However, we could keep the original react-vr intact; then this becomes a semver-minor change instead of a major.

**Comment by kkemple** [comment]
Looking at https://github.com/lelandrichardson/react-primitives/blob/master/src/index.vr.js#L1

I can't see how to make the split between 360 and vr since injection script is required in `index.vr.js` which will load for either platform unless `index.360.js` is supported and we can add a new root entry point. Investigating that now.

**Comment by ljharb** [comment]
yes, that's what i was thinking - adding a different entry point for 360, and leaving the vr one for vr.

**Comment by ljharb** [comment]
Ah, gotcha.

Ok, what about this: do a conditional try/catch require - ie, try 360 first, then fall back to vr?

**Comment by mathieudutour** [comment]
a try/catch is going to lead to warnings in a webpack compilation so I'm going to merge this and publish it as a breaking change

**Pull request merged**